h2o-automl/src/main/java/water/automl/api/schemas3/EventLogV99.java

```diff
@@ -11,15 +12,18 @@
   @API(help="List of events produced during the AutoML run", direction=API.Direction.OUTPUT)
   public EventLogEntryV99[] events;
 
+  @API(help="A table representation of this event log, for easy rendering", direction=API.Direction.OUTPUT)
+  public TwoDimTableV3 table;
```

Comment:
@seb-h2o does EventLogs have anything to do with leaderboard extensions?

Reply:
No. As explained in the `refactoring` section of the ticket description. I had to move `Leaderboard` to its package to isolate the leaderboard logic.
As there is some kind of structural symmetry between EventLog and Leaderboard design, I also moved the first one to its package, and applied the same small changes at API level.